test/e2e/node_drain.go

```diff
@@ -575,6 +568,20 @@ func NodeDrainTimeoutSpec(ctx context.Context, inputGetter func() NodeDrainTimeo
 			input.UnblockNodeVolumeDetachment(ctx, input.BootstrapClusterProxy, cluster)
 		}
 
```

Comment:
Let's add a short godoc to explan that we do this so the drain of the 2nd CP machine doesn't get stuck